monitoring/rid_qualifier/host/forms.py

```diff
@@ -0,0 +1,20 @@
+import json
+from flask_wtf import FlaskForm
+from wtforms import StringField, SubmitField, TextAreaField
+from wtforms.validators import DataRequired, ValidationError
+
+
+class UserConfig(FlaskForm):
+    auth_spec = StringField('Auth Spec', validators=[DataRequired()])
+    user_config = TextAreaField('User Config', validators=[DataRequired()])
+    submit = SubmitField('Submit')
+
+    def validate_user_config(form, field):
+        try:
+            user_config = json.loads(field.data)
+        except ValueError as e:
+            raise ValueError(e)
```

Comment:
If the only thing we do here in case of a ValueError during `json.loads` is re-raise the error, we can just drop the try block and allow `json.loads` to raise the original error uncaught.